main/src/ui/util/helper.vala

```diff
@@ -225,6 +225,27 @@ public static string parse_add_markup(string s_, string? highlight_word, bool pa
                 assert_not_reached();
             }
         }
+
+        // Quotes are handled differently, since `>` is not matched
+        try {
+            // Match a whole line only
+            Regex regex = new Regex("(?:^|\n)(&gt; ?)(.+(?:$|\n))");
```

Comment:
Maybe any XMPP expert can confirm if `>` characters are always going to be encoded as `&gt;`?

Reply:
I think it is done by GTK then, because it certainly doesn't work if I use a literal `>`. Maybe related to the pango syntax they use?